xbmc/dbwrappers/Database.h

```diff
@@ -255,7 +255,13 @@ class CDatabase
                         CDbUrl& dbUrl,
                         SortDescription& sorting);
 
-  bool Connect(const std::string& dbName, const DatabaseSettings& db, bool create);
+  enum class ConnectionState
+  {
+    STATE_ERROR,
+    STATE_DATABASE_NOT_FOUND,
+    STATE_CONNECTED,
+  };
```

Comment:
newline to separate from next line please

Reply:
Will change that.